src/blocks/content-helper/content-helper-provider.ts

```diff
@@ -80,7 +83,7 @@ class ContentHelperProvider {
 			return Promise.reject( error );
 		}
 
-		let message = `${ __( 'Top-performing posts', 'wp-parsely' ) } ${ apiQuery.message }.`;
+		let message = `${ __( 'Top-performing posts', 'wp-parsely' ) } ${ apiQuery.message } ${ __( `in last`, 'wp-parsely' ) } ${ RELATED_POSTS_DEFAULT_TIME_RANGE } ${ __( `days.`, 'wp-parsely' ) }`;
```

Comment:
Instead of three separate strings:
- `Top-performing posts`
- `in last`
- `days.`
... this should be a single string with placeholders and values that are sprintf'd in as needed. This makes it a far easier string for translators to translate correctly.

For PHP, that would be something like
- `Top-performing posts %s in last %s days.`